NetKAN/CustomAsteroids-Pops-Stock-Stockalike.netkan

```diff
@@ -0,0 +1,42 @@
+{
+    "spec_version": 1,
+    "identifier": "CustomAsteroids-Pops-Stock-Stockalike",
+    "name": "Custom Asteroids (stockalike config)",
+    "abstract": "Emulates stock asteroids with Custom Asteroids groups.",
+    "$kref": "#/ckan/spacedock/210",
+    "$vref" : "#/ckan/ksp-avc",
+    "x_netkan_license_ok" : true,
+    "depends": [
+        {
+            "name": "CustomAsteroids",
+            "min_version": "v1.3.0",
+            "max_version": "v1.99.99",
+            "comment": "Requires intercept blocks and conditional spawning."
+        }
+    ],
+    "provides": [ "CustomAsteroids-Pops" ],
+    "conflicts": [
+        {
+            "name": "RealSolarSystem",
+            "comment": "The 6.4× and 10× Kerbol packs are compatible; how to allow them?"
```

Comment:
Every 3 hours @netkan-bot inflates all .netkans in the repo and if there are differences between the .ckans generated vs the ones already present, it commits them to ckan-meta. So conflict changes will be absolutely be passed on.

A caveat to note; .netkans only inflate metadata for the most recent release of a mod, to change metadata for previous versions manual edits to the .ckans must be made (typically not an issue as users like to stay up to date).